src/design/optForceGAMS/optForceWithGAMS.m

```diff
@@ -0,0 +1,840 @@
+%
+% targetRxn (obligatory):   Type: string
+%                           Description: string containing the ID for the
+%                           reaction whose flux is intented to be increased.
+%                           For example, if the production of succionate is
+%                           desired to be increased, 'EX_suc' should be
+%                           chosen as the target reaction
+%                           Example: targetRxn='EX_suc';
+%
+% mustU (obligatory):       Type: cell array.
+%                           Description: List of reactions in the MustU set
+%                           This input can be obtained by running the
+%                           script findMustU.m
+%                           Alternatively, there is a second usage of this
+%                           input:
+%                           Type: string.
+%                           Description: name of the .xls file containing
+%                           the list of the reactions in the MustU set
+%                           Example first usage: mustU={'R21_f';'R22_f'};
+%                           Example second usage: mustU='MustU';
+%
+% mustL (obligatory):       Type: cell array.
+%                           Description: List of reactions in the MustL set
+%                           This input can be obtained by running the
+%                           script findMustL.m
+%                           Alternatively, there is a second usage of this
+%                           input:
+%                           Type: string.
+%                           Description: name of the .xls file containing
+%                           the list of the reactions in the MustU set
+%                           Example first usage: mustL={'R11_f';'R26_f'};
+%                           Example second usage: mustL='MustL';
+%
+% minFluxesW (obligatory):   Type: double array of size n_rxns x1
+%                            Description: Minimum fluxes for each reaction
+%                            in the model for wild-type strain
+%                            Example: minFluxesW=[-90; -56];
+%
+% maxFluxesW (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Maximum fluxes for each reaction
+%                            in the model for wild-type strain
+%                            Example: maxFluxesW=[92; -86];
+%
+% minFluxesM (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Minimum fluxes for each reaction
+%                            in the model for mutant strain
+%                            Example: minFluxesW=[-90; -56];
+%
+% maxFluxesM (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Maxmum fluxes for each reaction
+%                            in the model for mutant strain
+%                            Example: maxFluxesW=[92; -86];
+%
+% k(optional):              Type: double
```

Comment:
for more than a few optional arguments, It is highly recommended to use an input parser and allow parameters as Parameter/Value pairs. Otherwise code with many input arguments is prone to typos, which worst case lead to wrong input being used unnoticed.
Also: There is a 
OPTIONAL INPUT: 
flag to indicate optional input for the function that is used in the documentation generation.  (in general please follow the documentation [guideline](https://opencobra.github.io/cobratoolbox/docs/documentationGuide.html)